pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/guide.en-us.md

```diff
@@ -0,0 +1,1275 @@
+csi-cinder-high-speed       cinder.csi.openstack.org   Delete          Immediate           true                   3d
+csi-hostpath-sc (default)   hostpath.csi.k8s.io        Retain          Immediate           false                  2d
+```
+
+## Step 1 - Installing TrilioVault for Kubernetes
+
+In this step, you will learn how to deploy `TrilioVault` for `OVH Managed Kubernetes Cluster`, and manage `TVK` installations via `Helm`. Backups data will be stored in the `OVH Object Storage` bucket created earlier in the [Prerequisites](#prerequisites) section.
+
+`TrilioVault` application can be installed as below:
+
+- The `TrilioVault Operator` (installable via `Helm`). You define a `TrilioVaultManager` CRD, which tells `TrilioVault` operator how to handle the `installation`, `post-configuration` steps, and future `upgrades` of the `Trilio` application components.
+- The `TrilioVault Manager` via the [k8s-triliovault] yaml definition (https://docs.trilio.io/kubernetes/use-triliovault/installing-triliovault#upstream-kubernetes)
+	
+
+### Installing TrilioVault Operator and Manager Using Helm
+
+**Important note**:
+
+`OVH Docs` tutorial is using the `Cluster` installation type for the `TVK` application (`applicationScope` Helm value is set to `"Cluster"`). All examples from this tutorial rely on this type of installation to function properly.
+
+Please follow the steps below, to install `TrilioVault` via `Helm`:
+
+1. First, clone the `OVH Docs` Git repository and change directory to your local copy:
+
+    ```shell
+    git clone https://github.com/ovh/docs.git
+
+    cd docs/pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/
+    ```
+
+2. Next, add the `TrilioVault` Helm repository, and list the available charts:
+
+    ```shell
+    helm repo add triliovault http://charts.k8strilio.net/trilio-stable/k8s-triliovault
+
+    helm search repo triliovault
+    ```
+
+    The output looks similar to the following:
+
+    ```text
+    NAME                                            CHART VERSION   APP VERSION     DESCRIPTION
+    triliovault-operator/k8s-triliovault-operator   2.6.4           2.6.4           K8s-TrilioVault-Operator is an operator designe...
+    triliovault/k8s-triliovault                     2.6.4           2.6.4           K8s-TrilioVault provides data protection and re...
+
+    ```
+    
+3. The chart of interest is `triliovault-operator/k8s-triliovault-operator`, which will install `TrilioVault for Kubernetes Operator` on the cluster. You can run `helm install` command to install the Operator. Install `TrilioVault for Kubernetes Operator` using `Helm`:
+
+    ```shell
+    TRILIOVAULT_CHART_VERSION="2.6.4"
+
+    helm install triliovault-operator triliovault-operator/k8s-triliovault-operator --version "${TRILIOVAULT_CHART_VERSION}" \
+      --namespace tvk \
+      --create-namespace
+    ```
+
```

Comment:
$ helm install triliovault-operator triliovault/k8s-triliovault-operator --version "${TRILIOVAULT_CHART_VERSION}"   --namespace tvk   --create-namespace
Error: INSTALLATION FAILED: failed to install CRD crds/triliovault.trilio.io_triliovaultmanagers.yaml: unable to recognize "": no matches for kind "CustomResourceDefinition" in version "apiextensions.k8s.io/v1beta1"

Reply:
@raclepoulpe I am guessing, this has happened because I missed adding another `helm chart repo` in above step 2.
I have corrected it now and added another repo `triliovault-operator` which will be used for the Operator installation.
Please check and let me know if this works at your end.